app.R

```diff
@@ -10,35 +10,89 @@ library(rmarkdown)
 library(rprojroot)
 source("src/r/model.R")
 
-ui <- shinyUI(fluidPage(
-  fluidRow(column(1),
-           column(10, h1("Prevalence of Common Mental Health Disorders among people aged 16 to 74,\n in England, by NHS Region, 2014/15")),
-            column(1)),
-  fluidRow(column(1),
-           column(10, sidebarPanel( 
+format_tab <- function(title, header, region_no, map_no, chart_no, narrative_no, metadata_url_no) {
+  tabPanel(title, 
```

Comment:
I would like the tab titles to be in a larger font (both to fit in with the rest of the page and for easy readability).